source/lib/src/cuda/gelu.cu

```diff
@@ -1,6 +1,9 @@
 #include "gelu.h"
 #include "device.h"
 
+__device__ inline double _tanh(double x) {return tanh(x);}
+__device__ inline float _tanh(float x) {return tanh(x);}
```

Comment:
Would it be better to use tanhf for a float input x ?
```suggestion
__device__ inline float _tanh(float x) {return tanhf(x);}
```

Reply:
Yes, it's a mistake.